ddtrace/appsec/_handlers.py

```diff
@@ -376,6 +380,9 @@ def _custom_protobuf_getattribute(self, name):
             source_origin=OriginType.GRPC_BODY,
         )
     elif isinstance(ret, MutableMapping):
+        if MessageMapContainer is None:
+            return ret
+
```

Comment:
```suggestion
    elif MessageMapContainer is not None, and isinstance(ret, MutableMapping):
```

don't feel like you have to accept this, this is a pedantic performance related improvement.

if in failure scenarios we can avoid the `isinstance` call it'll help with performance.

again, pedantic, feel free to just resolve.

Reply:
I'd accept if you remove the `,` before the `and`

Reply:
updated 🤦🏻